hadoop-hdds/server-scm/pom.xml

```diff
@@ -48,6 +48,11 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-hdds-client</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-ozone-common</artifactId>
+      <version>1.1.0-SNAPSHOT</version>
```

Comment:
It is wired to me that my local checkstyle contains when there is no `version` in here.